Undefined reference to DynamicFunc__* without Perl

```
`/usr/lib/gcc/x86_64-pc-cygwin/11/../../../../x86_64-pc-cygwin/bin/ld: dynamic_preloads.o:/cygdrive/c/cygwin64/john-bleeding-jumbo/src/dynamic_preloads.c:3168: undefined reference to `DynamicFunc__KECCAK_384_crypt_input1_overwrite_input2'
/usr/lib/gcc/x86_64-pc-cygwin/11/../../../../x86_64-pc-cygwin/bin/ld: dynamic_preloads.o:dynamic_preloa:(.data+0x7108): undefined reference to `DynamicFunc__KECCAK_384_crypt_input2_to_output1_FINAL'
/usr/lib/gcc/x86_64-pc-cygwin/11/../../../../x86_64-pc-cygwin/bin/ld: dynamic_preloads.o:dynamic_preloa:(.data+0x7138): undefined reference to `DynamicFunc__KECCAK_384_crypt_input1_to_output1_FINAL'
/usr/lib/gcc/x86_64-pc-cygwin/11/../../../../x86_64-pc-cygwin/bin/ld: dynamic_preloads.o:dynamic_preloa:(.data+0x7178): undefined reference to `DynamicFunc__KECCAK_384_crypt_input1_to_output1_FINAL'
/usr/lib/gcc/x86_64-pc-cygwin/11/../../../../x86_64-pc-cygwin/bin/ld: dynamic_preloads.o:/cygdrive/c/cygwin64/john-bleeding-jumbo/src/dynamic_preloads.c:3168: undefined reference to `DynamicFunc__KECCAK_384_crypt_input1_to_output1_FINAL'
/usr/lib/gcc/x86_64-pc-cygwin/11/../../../../x86_64-pc-cygwin/bin/ld: dynamic_preloads.o:dynamic_preloa:(.data+0x7f28): undefined reference to `DynamicFunc__KECCAK_224_crypt_input1_append_input2'
/usr/lib/gcc/x86_64-pc-cygwin/11/../../../../x86_64-pc-cygwin/bin/ld: dynamic_preloads.o:dynamic_preloa:(.data+0x7f38): undefined reference to `DynamicFunc__KECCAK_224_crypt_input2_to_output1_FINAL'
/usr/lib/gcc/x86_64-pc-cygwin/11/../../../../x86_64-pc-cygwin/bin/ld: dynamic_preloads.o:dynamic_preloa:(.data+0x7f78): undefined reference to `DynamicFunc__KECCAK_224_crypt_input2_to_output1_FINAL'
/usr/lib/gcc/x86_64-pc-cygwin/11/../../../../x86_64-pc-cygwin/bin/ld: dynamic_preloads.o:dynamic_preloa:(.data+0x7fb8): undefined reference to `DynamicFunc__KECCAK_224_crypt_input2_to_output1_FINAL'
/usr/lib/gcc/x86_64-pc-cygwin/11/../../../../x86_64-pc-cygwin/bin/ld: dynamic_preloads.o:dynamic_preloa:(.data+0x7ff0): undefined reference to `DynamicFunc__KECCAK_224_crypt_input1_to_output1_FINAL'
/usr/lib/gcc/x86_64-pc-cygwin/11/../../../../x86_64-pc-cygwin/bin/ld: dynamic_preloads.o:dynamic_preloa:(.data+0x8008): undefined reference to `DynamicFunc__KECCAK_224_crypt_input1_overwrite_input2'
/usr/lib/gcc/x86_64-pc-cygwin/11/../../../../x86_64-pc-cygwin/bin/ld: dynamic_preloads.o:dynamic_preloa:(.data+0x8010): undefined reference to `DynamicFunc__KECCAK_224_crypt_input2_to_output1_FINAL'
/usr/lib/gcc/x86_64-pc-cygwin/11/../../../../x86_64-pc-cygwin/bin/ld: dynamic_preloads.o:/cygdrive/c/cygwin64/john-bleeding-jumbo/src/dynamic_preloads.c:3120: undefined reference to `DynamicFunc__KECCAK_224_crypt_input1_overwrite_input2'
/usr/lib/gcc/x86_64-pc-cygwin/11/../../../../x86_64-pc-cygwin/bin/ld: dynamic_preloads.o:dynamic_preloa:(.data+0x8028): undefined reference to `DynamicFunc__KECCAK_224_crypt_input2_to_output1_FINAL'
/usr/lib/gcc/x86_64-pc-cygwin/11/../../../../x86_64-pc-cygwin/bin/ld: dynamic_preloads.o:dynamic_preloa:(.data+0x8058): undefined reference to `DynamicFunc__KECCAK_224_crypt_input1_to_output1_FINAL'
/usr/lib/gcc/x86_64-pc-cygwin/11/../../../../x86_64-pc-cygwin/bin/ld: dynamic_preloads.o:dynamic_preloa:(.data+0x8098): undefined reference to `DynamicFunc__KECCAK_224_crypt_input1_to_output1_FINAL'
/usr/lib/gcc/x86_64-pc-cygwin/11/../../../../x86_64-pc-cygwin/bin/ld: dynamic_preloads.o:/cygdrive/c/cygwin64/john-bleeding-jumbo/src/dynamic_preloads.c:3120: undefined reference to `DynamicFunc__KECCAK_224_crypt_input1_to_output1_FINAL'
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:1462: ../run/john.exe] Error 1
make: *** [Makefile:190: default] Error 2
`
```
Windows 10
I suspect there is no package, but there is no link in the description

`make distclean && ./configure && make -s clean && make -sj4`


```
zed_fmt_plug.c:135:25: note: referencing argument 4 of type ‘const unsigned char **’
zed_fmt_plug.c:135:25: warning: ‘pkcs12_pbe_derive_key_simd’ accessing 26880 bytes in a region of size 32 [-Wstringop-overflow=]
zed_fmt_plug.c:135:25: note: referencing argument 5 of type ‘size_t *’ {aka ‘long unsigned int *’}
zed_fmt_plug.c:135:25: warning: ‘pkcs12_pbe_derive_key_simd’ accessing 26880 bytes in a region of size 32 [-Wstringop-overflow=]
zed_fmt_plug.c:135:25: note: referencing argument 8 of type ‘unsigned char **’
In file included from zed_fmt_plug.c:30:
pkcs12.h:54:5: note: in a call to function ‘pkcs12_pbe_derive_key_simd’
   54 | int pkcs12_pbe_derive_key_simd( int md_type, int iterations, int id, const unsigned char *pwd[SIMD_MAX_GROUP_PFX],
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~
Securing executables.

Make process completed.
```

**All Ok**

--- a/src/configure
+++ b/src/configure
@@ -7780,6 +7780,12 @@ $as_echo "no" >&6; }
 fi
 
 
+if test "x$PERL" == x ; then
+  { { $as_echo "$as_me:${as_lineno-$LINENO}: error: in \`$ac_pwd':" >&5
+$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
+as_fn_error $? "No perl executable found
+See \`config.log' for more details" "$LINENO" 5; }
+fi
 if test -z "$AS"; then :
   AS="$CC"
 fi

--- a/src/configure.ac
+++ b/src/configure.ac
@@ -339,6 +339,9 @@ AC_CACHE_SAVE
 AC_PATH_PROG([SORT], [sort])
 AC_PATH_PROG([FIND], [find])
 AC_PATH_PROG([PERL], [perl])
+if test "x$PERL" == x ; then
+  AC_MSG_FAILURE([No perl executable found])
+fi
 AS_IF([test -z "$AS"], [AS="$CC"])
 AS_IF([test -z "$LD"], [LD="$CC"])
 AS_IF([test -z "$AR"], [AC_CHECK_TOOL([AR], [ar])])

